src/components/views/right_panel/RoomSummaryCard.tsx

```diff
@@ -199,6 +200,8 @@ const RoomSummaryCard: React.FC<IProps> = ({ room, onClose }) => {
     };
 
     const isRoomEncrypted = useIsEncrypted(cli, room);
+    const roomContext = useContext(RoomContext);
+    const e2eStatus = roomContext['e2eStatus'];
```

Comment:
```suggestion
    const e2eStatus = roomContext.e2eStatus;
```

Reply:
We don't have types for this, so it will fail the linter.

Reply:
(for context, because RoomContext has been TSified on develop but not in the RC)